critiquebrainz/frontend/views/review.py

```diff
@@ -382,6 +383,6 @@ def hide(id):
 @admin_view
 def unhide(id):
     review = db_review.get_or_404(id)
-    db_review.unhide(review["id"])
+    db_review.set_hidden_state(review["id"], is_hidden="False")
```

Comment:
If you have to accept boolean arguments as strings in an interface, it's a good sign that something is not right. :)